docs/package.json

```
@@ -9,7 +9,7 @@
     "build:clean": "rimraf .next && yarn build",
     "build-sw": "node ./scripts/buildServiceWorker.js",
     "dev": "rimraf ./node_modules/.cache/babel-loader && next dev --port 3001",
-    "deploy": "git push upstream master:release",
+    "deploy": "git push upstream next:release",
```

I assume we need a new branch for the docs of for v5:
```suggestion
    "deploy": "git push upstream next:docs-v5",
```